operations/workplace/people/performance-reviews-50/README.md

```diff
@@ -4,38 +4,42 @@ description: How to do performance reviews the Mattermost way.
 
 # Performance Reviews
 
+The performance review process is a vital practice to reflect on the output of ourselves and our colleagues, to identify the key opportunities for each of us to grow and excel, to objectively review our capabilities and track record on delivering results against the requirements of our role levels, which may result in level-based promotions, compensation adjustments, and/or changes in role. 
+
 ## Why are performance reviews important? 
 
 The responsibility of a manager is to increase the total output of the organizations they control and the organizations they influence. 
 
-The purpose of a performance review is to assess a colleague's performance, have the colleague clearly understand how to progress professionally in the opinion of the manager, and hold the colleague accountable for increasing their performance based on the assessment and feedback. 
+The primary purpose of a performance review is to assess a colleague's performance, have the colleague clearly understand how to progress professionally in the opinion of the manager, and hold the colleague accountable for increasing their performance based on the assessment and feedback. 
 
 It is one of the most difficult things a people manager needs to do, it is also one of the most important. 
 
 ### Other than increasing output, what are other objectives of performance reviews? 
 
-The primary goal of performance reviews is to increase the output of colleagues and of the organization. 
+In addition to the primary goal increasing the output of colleagues and of the organization, the performance review process also serves to align our actions to [Mattermost Leadership Principles](../../../../company/about-mattermost/#leadership-principles), which are used as the criteria for peer-evaluation, and to calibrate role levels with a colleague's, which may result in level-based promotions and compensation adjustments or if level expectations are repeatedly not met, changes in role. 
+
+## Manager Training 
+
+We believe a manager’s fundamental purpose of a performance review is to improve performance. We use Chapter 13 of the book [High Output Management](https://www.amazon.com/High-Output-Management-Andrew-Grove-ebook/dp/B015VACHOK) as the framework and pre-reading for our review process. 
 
-The process also serves to align our actions to [Mattermost Leadership Principles](../../../../company/about-mattermost/#leadership-principles), which are used as the criteria for peer-evaluation, and to calibrate role levels with a colleague's, which may result in level-based promotions and compensation adjustments or if level expectations are repeatedly not met, changes in role. 
+At performance review time all managers should: 
 
-## **High Output Management**
+1. Read this document and the related documents to their departmental review process in detail. 
```

Comment:
@it33 - is there supposed to be a document linked to "this document" or is that referencing this whole handbook page?

Reply:
Yes, I've updated to make it more clear. Thanks for the question.